docs/bundler/migration.md

```diff
@@ -0,0 +1,1107 @@
+- Bun uses `jsxDEV` by default. Set `NODE_ENV=production` to use the production `jsx` transform instead.
+
+---
+
+- `--jsx-factory`
+- `--jsx-factory`
+
+---
+
+- `--jsx-fragment`
+- `--jsx-fragment`
+
+---
+
+- `--jsx-import-source`
+- `--jsx-import-source`
+
+---
+
+- `--jsx-side-effects`
+- n/a
+- JSX is always assumed to be side-effect-free.
+
+---
+
+- `--keep-names`
+- n/a
+- Not supported
+
+---
+
+- `--keyfile`
+- n/a
+- Not applicable
+
+---
+
+- `--legal-comments`
+- n/a
+- Not supported
+
+---
+
+- `--log-level`
+- n/a
+- Not supported. This can be set in `bunfig.toml` as `logLevel`.
+
+---
+
+- `--log-limit`
+- n/a
+- Not supported
+
+---
+
+- `--log-override:X=Y`
+- n/a
+- Not supported
+
+---
+
+- `--main-fields`
+- n/a
+- Not supported
+
+---
+
+- `--mangle-cache`
+- n/a
+- Not supported
+
+---
+
+- `--mangle-props`
+- n/a
+- Not supported
+
+---
+
+- `--mangle-quoted`
+- n/a
+- Not supported
+
+---
+
+- `--metafile`
+- `--manifest`
+
+---
+
+- `--minify-whitespace`
+- `--minify-whitespace`
+
+---
+
+- `--minify-identifiers`
+- `--minify-identifiers`
+
+---
+
+- `--minify-syntax`
+- `--minify-syntax`
+
+---
+
+- `--out-extension`
+- n/a
+- Not supported
+
+---
+
+- `--outbase`
+- `--root`
+- Not supported
+
+---
+
+- `--preserve-symlinks`
+- n/a
+- Not supported
+
+---
+
+- `--public-path`
+- `--public-path`
+
+---
+
+- `--pure`
+- n/a
+- Not supported
+
+---
+
+- `--reserve-props`
+- n/a
+- Not supported
+
+---
+
+- `--resolve-extensions`
+- n/a
+- Not supported
+
+---
+
+- `--servedir`
+- n/a
+- Not applicable
+
+---
+
+- `--source-root`
+- n/a
+- Not supported
+
+---
+
+- `--sourcefile`
+- n/a
+- Not supported. Bun does not support `stdin` input yet.
+
+---
+
+- `--sourcemap`
+- `--sourcemap`
+- No differences
+
+---
+
+- `--sources-content`
+- n/a
+- Not supported
+
+---
+
+- `--supported`
+- n/a
+- Not supported
+
+---
+
+- `--tree-shaking`
+- n/a
+- Always `true`
+
+---
+
+- `--tsconfig`
+- `--tsconfig-override`
+
+---
+
+- `--version`
+- n/a
+- Run `bun --version` to see the version of Bun.
+
+{% /table %}
+
+## JavaScript API
+
+{% table %}
+
+- `esbuild.build()`
+- `Bun.build()`
+
+---
+
+- `absWorkingDir`
+- n/a
+- Always set to `process.cwd()`
+
+---
+
+- `alias`
+- n/a
+- Not supported
+
+---
+
+- `allowOverwrite`
+- n/a
+- Always `true`
+
+---
+
+- `assetNames`
+- `naming.asset`
+- Uses same templating syntax as esbuild, but `[ext]` must be included explicitly.
+
+  ```ts
+  Bun.build({
+    entrypoints: ["./index.tsx"],
+    naming: {
+      asset: "[name].[ext]",
+    },
+  });
+  ```
+
+---
+
+- `banner`
+- n/a
+- Not supported
+
+---
+
+- `bundle`
+- n/a
+- Always `true`
+
+---
+
+- `charset`
+- n/a
+- Not supported
+
+---
+
+- `chunkNames`
+- `naming.chunk`
+- Uses same templating syntax as esbuild, but `[ext]` must be included explicitly.
+
+  ```ts
+  Bun.build({
+    entrypoints: ["./index.tsx"],
+    naming: {
+      chunk: "[name].[ext]",
+    },
+  });
+  ```
+
+---
+
+- `color`
+- n/a
+- Always `true`
```

Comment:
should be N/A since we do not do any logging, but instead hand off an error object.